client/checkout/woopay/direct-checkout/woopay-direct-checkout.js

```diff
@@ -234,6 +279,51 @@ class WooPayDirectCheckout {
 		);
 	}
 
+	/**
+	 * Gets the WooPay redirect data.
+	 *
+	 * @return {Promise<Promise<*>|*>} Resolves to the WooPay redirect response.
+	 */
+	static async getWooPayMinimumSesssionDataFromMerchant() {
+		if ( getConfig( 'woopayMinimumSessionData' ) ) {
+			return getConfig( 'woopayMinimumSessionData' );
+		}
```

Comment:
Yeah, it should always be defined. I was just being extra cautious here with a request fallback in case of some edge cases. I've added a comment in https://github.com/Automattic/woocommerce-payments/pull/8268/commits/416f4d9c49cfe93f2999ab0f71becf4e3298862a to clarify that